### ISSUE TITLE
HrOrgChartOverview

### ISSUE THREAD

**Issue by YanShaolong** [OCA/hr#890]
module: hr_org_chart_overview
version: 13.0

**Steps to reproduce**
- Install to Odoo 13 Community
- Generate report from: Employees-Reporting

**Current behavior**
 Error message: 
  Error occurs:
Traceback:
Error: QWeb2: Template 'HrOrgChartOverview' not found
    at Object.exception (http://localhost:8070/web/content/884-4a1fe58/web.assets_common.js:3692:7)
    at Engine._render (http://localhost:8070/web/content/884-4a1fe58/web.assets_common.js:3737:103)
    at Engine.render (http://localhost:8070/web/content/884-4a1fe58/web.assets_common.js:3733:151)
    at Class.renderElement (http://localhost:8070/web/content/945-c069cdc/web.assets_backend.js:471:282)
    at Class.prototype.<computed> [as renderElement] (http://localhost:8070/web/content/884-4a1fe58/web.assets_common.js:3860:488)
    at http://localhost:8070/web/content/884-4a1fe58/web.assets_common.js:4100:6
**Expected behavior**
  No error, but the org chart displaying

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. Are you able to reproduce the problem in runbot infrastructure (https://runbot.odoo-community.org/runbot/116/13.0 - click on the blue door icon of the version row - admin/admin)?

If so, please explain with all the possible detailed steps to get to the error. Not having them, it's very difficult to hunt down the problem. Please don't be shy of adding details, and even better if you provide some screenshots or videos.

If not, this can be due to a configuration change, a custom/community module you have deployed in your DB or an outdated code. Please report back if it's the case.

For keeping a clean house in this issue tracker, if you don't answer in several days, this will be closed assuming it was one of the stated things.

**Comment by YanShaolong** [comment]
Hi Pedro,

Thanks for the reply.
You are right, that I tested again on version 13.0-20191010 which is
working fine. The error I reported was on Odoo 13.0-20200827.
I tested in that runbot environment (Odoo 14.1alpha1), the different error
as below:
Steps:
- download code from Github
- put it in a zip named hr_org_chart_overview.zip
- log on to the runbot environment
- Activate developer mode
- Import module (the zip file above), and install it
- Open the Org Chart from Employees/Reporting/Organizational Chart
- The below error occurred

[image: image.png]

Regards,
Shaolong
